src/index.js

```diff
@@ -330,11 +340,8 @@ class App extends React.Component {
               if (this.state.elementType === "selection") {
                 if (isDraggingElements) {
                   isDraggingElements = false;
-                } else {
-                  // Remove actual selection element
-                  setSelection(draggingElement);
-                }
-                elements.pop();
```

Comment:
How is the selection background removed? Can’t find any replacement.

To repro: mouse down, drag, mouse up. The blue rectangle should be gone

Reply:
I am checking it, I had a conflict